getdata_HN.py

```diff
@@ -1,57 +1,61 @@
-import requests
+#!/usr/bin/env python
 import json
 import os.path
+import requests
 from urllib.parse import urljoin
 
 URL = 'https://hacker-news.firebaseio.com/v0/'
 KEYWORDS = ['python', 'django']
-story = {}
+stories = {}
 
 
-def get_json(param):
+def _get_json(param):
     url = urljoin(URL, param)
-    req = requests.get(url)
-    return req.json()
+    res = requests.get(url)
+    return res.json()
 
 
-def get_stories():
+def _get_stories():
     parameter = 'topstories.json?print=pretty'
-    return get_json(parameter)
+    return _get_json(parameter)
 
 
 def check_item():
-    for item in get_stories():
+    for item in _get_stories():
         parameter = 'item/{}.json?print=pretty'.format(item)
-        item = get_json(parameter)
+        item = _get_json(parameter)
         for kwd in KEYWORDS:
             if kwd in item['title']:
                 return item
 
 
-def get_item(item):
+def _get_item(item):
     parameter = 'item/{}.json?print=pretty'.format(item)
-    item = get_json(parameter)
-    story[item['id']] = {'title': item['title']}
+    item = _get_json(parameter)
+    stories[item['id']] = {'title': item['title']}
     return
 
 
-def save_items(items, filename='tmp-data.json'):
+def _save_items(items, filename='data.json'):
```

Comment:
data.json is everywhere in your code. You should set a name for it and use the name instead